yt_dlp/extractor/mxplayer.py

```diff
@@ -28,53 +29,54 @@ class MxplayerIE(InfoExtractor):
         'url': 'https://www.mxplayer.in/movie/watch-knock-knock-hindi-dubbed-movie-online-b9fa28df3bfb8758874735bbd7d2655a?watch=true',
         'info_dict': {
             'id': 'b9fa28df3bfb8758874735bbd7d2655a',
+            'display_id': 'episode-1-online',
             'ext': 'mp4',
             'title': 'Knock Knock (Hindi Dubbed)',
-            'description': 'md5:b195ba93ff1987309cfa58e2839d2a5b',
-            'season_number': 0,
-            'episode_number': 0,
+            'description': 'Evan, a devoted husband, and father, invites double trouble as he allows two girls to seek shelter at his place. His sweet gesture takes the shape of a dangerous cat and mouse game.',
+            'season_number': None,
+            'episode_number': None,
```

Comment:
If the value is None, you don't need to add that key to the test. I think... mmm

Reply:
Yes, don't add None to tests

Reply:
Got it. Done.